### PR TITLE
Add inspection utility methods to Env and Key

### PR DESCRIPTION
**Pull request by Zarenor**
These are intended for debugging and theme manipulation, not general use.

### CODE REVIEW

File: druid/src/env.rs [hunk 1]
```diff
@@ -208,6 +208,25 @@ impl Env {
             .map(|value| value.to_inner_unchecked())
     }
 
+    /// Gets a value from the environment, in it's encapsulated [`Value`] form,
```
Comment (xStrom) [inline druid/src/env.rs:211]: ~it's~ its

File: druid/src/env.rs [hunk 2]
```diff
@@ -208,6 +208,25 @@ impl Env {
...
+    pub fn get_untyped(&self, key: impl Borrow<Key<()>>) -> &Value {
+        let key = key.borrow();
+        if let Some(value) = self.0.map.get(key.key) {
+            value
+        } else {
+            panic!("key for {} not found", key.key)
+        }
+    }
+
+    pub fn try_get_untyped(&self, key: impl Borrow<Key<()>>) -> Option<&Value> {
```
Comment (xStrom) [inline druid/src/env.rs:226]: Let's document, even if briefly, all `pub` functions.

File: druid/src/env.rs (hunk 1 above)
Comment (Zarenor) [reply druid/src/env.rs]: Silly inconsistent grammar in my native language 😬
Should be fixed.

File: druid/src/env.rs (hunk 2 above)
Comment (Zarenor) [reply druid/src/env.rs:232]: Thanks. Done.

File: druid/src/env.rs [hunk 3]
```diff
@@ -208,6 +208,39 @@ impl Env {
...
+    ///
+    /// *WARNING:* This is not intended for general use, but only for inspecting an `Env` e.g.
+    /// for debugging, theme editing, and theme loading.
+    /// [`Value`]: enum.Value.html
+    pub fn get_untyped(&self, key: impl Borrow<Key<()>>) -> &Value {
+        let key = key.borrow();
+        if let Some(value) = self.0.map.get(key.key) {
+            value
+        } else {
+            panic!("key for {} not found", key.key)
```
Comment (xStrom) [inline druid/src/env.rs:222]: Let's document panic conditions under the *Panics* header. Take a look at some of the other functions here like `try_get`.

File: druid/src/env.rs [hunk 4]
```diff
@@ -208,6 +208,43 @@ impl Env {
...
+    /// [`Value`]: enum.Value.html
+    pub fn try_get_untyped(&self, key: impl Borrow<Key<()>>) -> Option<&Value> {
+        self.0.map.get(key.borrow().key)
+    }
+
+    /// Gets the entire contents of the `Env`, in key-value pairs.
+    ///
+    /// *WARNING:* This is not intended for general use, but only for inspecting an `Env` e.g.
+    /// for debugging, theme editing, and theme loading.
+    pub fn get_all(&self) -> impl ExactSizeIterator<Item = (&String, &Value)> {
```
Comment (cmyr) [inline druid/src/env.rs:244]: a little late to the game but I might've just called this `iter` or `iter_raw` or something. 🤷

Comment (Zarenor) [reply druid/src/env.rs:244]: Ah. Hm. That might have made some idiomatic sense. No reason not to change it. I'll try to find some time this weekend.

Comment (cmyr) [reply druid/src/env.rs:244]: no pressure, just an observation!

**Pull request merged**